### ISSUE TITLE
MP4 Video Output not working correctly on UDOO hardware. 

### ISSUE THREAD

**Issue by patdelaney** [FalconChristmas/fpp#1071]
**FPP Version and Hardware**: 
Example: *FPP 5.x running on UDOO*
Running DEBIAN 10

I'm not sure if we ever tested MP4 playback on a non PI device.
"Blank Screen on Startup" not available on Debian hardware device (UDOO)
Force HDMI settings or the availability
![20210809_221306](https://user-images.githubusercontent.com/8147459/128802872-a9bd2410-50fc-4fba-9c32-583ef94de001.jpg)
 to select a resolution is not available for Non PI hardware.

See attached pic of HDMI output

**Comment by patdelaney** [comment]
https://www.udoo.org/discover-udoo-x86-ii/

**Comment by patdelaney** [comment]
Installing FPP on UDOO-x86 board
Install Latest Debian x64 build on to UDOO follow UDOO instructions on a bootable USB stick
I unchecked all extras so its a bare—bones install
I did have to run apt-get update and then apt-get install wget

I ran the FPP_Install.sh with the –img  flag

The FPP installed and I have the following observations:
•	The banner still shows 2 ip addresses probably related to the stock Debian installs a DHCP client and then we setup conman. (Perhaps we modify the script to remove the Debian installed DHCP client.
![image](https://user-images.githubusercontent.com/8147459/129459695-182452c5-9955-4bab-bc04-d435a32789e2.png)
 
•	The UDOO has wireless. I seem to recall that we need to setup wireless on non PI or BBB hardware  via the native operating system. Is that correct?
•	If I look at Interface settings, I can see the following:  (I’m assuming that wlp3s0 is the wireless interface.)  It gives us the ability to set Wi-Fi to Static, but not to actually setup the SSID via the UI.
 
![image](https://user-images.githubusercontent.com/8147459/129459700-1cc3e470-edde-4efc-95fc-ddb486af44df.png)


•	The UDOO has 3 video outputs. I’m not sure we can utilize them at this point.
•	I don’t see the ability to “blank screen after startup” in the UI so the console is always on the monitor.
•	Can’t play video clips with udoo  see bug  https://github.com/FalconChristmas/fpp/issues/1071
o	MP4 Video Output not working correctly on UDOO hardware
•	UDOO has the following output Mixer devices
 
![image](https://user-images.githubusercontent.com/8147459/129459704-9fdd618a-ae28-4543-adb5-771fe9a841cc.png)

•	Audio appears to play ok out the speaker mini jack
